test/Playground.js

```diff
@@ -16,49 +17,49 @@ describe("A playground", function() {
   });
 
   it("can list its codeObject's ids", function () {
-    playground.getOrCreateCodeObject("bob");
-    playground.getOrCreateCodeObject("jack");
-    expect(playground.population()).to.have.members(["bob","jack"]);
+    playground.getOrCreateCodeObject('bob');
+    playground.getOrCreateCodeObject('jack');
+    expect(playground.population()).to.have.members(['bob', 'jack']);
   });
 
-  it("can tell if it is empty", function () {
+  it('can tell if it is empty', function () {
```

Comment:
In tests I would prefer if double quotes would not be replaced by simple quotes. For example here, it has been replaced but not in the previous `it(` where their was a single quote in the text.

Reply:
Yeah, this isn't very pretty but there will always be less-than-ideal cases and I'd rather not spend time discussing each rule, that's the point of picking `semistandard`. Please see https://standardjs.com/#i-disagree-with-rule-x-can-you-change-it …

Reply:
Ok, let's move on then. 

Juste saying, I don't like the tone of the pitch behind the link. What is the point of being so agressive !